sensing/position_estimation.cpp

```diff
@@ -489,18 +474,6 @@ Position_estimation::fence_violation_state_t Position_estimation::get_fence_viol
 }
 
 
-void Position_estimation::set_new_fence_origin()
```

Comment:
local_position_t does not have an origin field, so the method is not needed anymore.